src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type CommentProp = {
+  comment: Comment,
+};
+
+export const CommentInfo: React.FC<CommentProp> = ({ comment }) => {
+  return (
+
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
+
+        {' by '}
+
+        <a
+          className="CommentInfo__email"
+          href={`mailto:${comment.email}`}
```

Comment:
If the email is undefined, we will get an error when clicking, try to predict it